game/scripts/npc/abilities/talents/alchemist_talent4.txt

```diff
@@ -7,19 +7,19 @@
   {
     // General
     //-------------------------------------------------------------------------------------------------------------
-    "ID"					"7054"														// unique ID number for this ability.  Do not change this once established or it will invalidate collected stats.
-    "AbilityType"					"DOTA_ABILITY_TYPE_ATTRIBUTES"
-    "AbilityBehavior"				"DOTA_ABILITY_BEHAVIOR_PASSIVE"
+    "ID"                                                  "7054"
+    "AbilityType"                                         "DOTA_ABILITY_TYPE_ATTRIBUTES"
+    "AbilityBehavior"                                     "DOTA_ABILITY_BEHAVIOR_PASSIVE"
 
     // Special
     //-------------------------------------------------------------------------------------------------------------
     "AbilitySpecial"
     {
       "01"
       {
-        "var_type"					"FIELD_INTEGER"
-        "value"				"80" //OAA
-        "ad_linked_ability"			"alchemist_chemical_rage"
+        "var_type"                                        "FIELD_INTEGER"
+        "value"                                           "50"
```

Comment:
really doesn't make a difference 30 hp regen is nothing in later stages of game and the nerfs should be targeted towards his late game dominance.